splash/tests/test_request_filters.py

```diff
@@ -155,3 +170,47 @@ def test_file_scheme_can_be_enabled(self):
 
         self.assertStatusCode(r, 200)
         self.assertIn('script.js', r.text)
+
+    def run_with_extra_args(self, extra_args): 
+        with SplashServer(extra_args=extra_args) as splash:
+            test_url = self.mockurl('subresources-with-caching')
+            render_url = splash.url('render.har')
+
+            # 1st page fetch
+            r = requests.get(render_url, params={'url': test_url, 'response_body': 1})
+            self.assertStatusCode(r, 200)
+            entry11 = self.findHAREntry(r.json(),r"^.*subresources-with-caching$")
+            entry12 = self.findHAREntry(r.json(),r"^.*subresources-with-caching/style.css$")
+            entry13 = self.findHAREntry(r.json(),r"^.*subresources-with-caching/img.gif$")
+
+            # 2nd page fetch
+            r = requests.get(render_url, params={'url': test_url, 'response_body': 1})
+            self.assertStatusCode(r, 200)
+            entry21 = self.findHAREntry(r.json(),r"^.*subresources-with-caching$")
+            entry22 = self.findHAREntry(r.json(),r"^.*subresources-with-caching/style.css$")
+            entry23 = self.findHAREntry(r.json(),r"^.*subresources-with-caching/img.gif$")
+
+            return [[entry11,entry12,entry13],[entry21,entry22,entry23]]
+
+    # run just this test using
+    # pytest -s --verbosity=6 ./splash/tests/test_request_filters.py -k test_disable_browser_caches
+    def test_disable_browser_caches(self):
+        # entries if run with caches enabled
+        # in this case we should have  [[content,content,content],[content,content,no content]]
```

Comment:
I wonder why isn't it working for css files. Maybe there are some other response headers needed to make it work, or maybe browser uses ETag or something like that, so there is no "full" response. It doesn't look good that opur code sets cache headers, but then checks that the resource is not cached; test will break if resource start to becached.

I think there are two ways to fix that:

1) Investigate what's going on and make sure css is cached as well;
2) remove a check for css file, check only html and gif.

Reply:
I agree with you on that and I went with the 2nd approach, I have removed the css checks. I've pushed a new commit for this.
For this pull request I'm ok with having it this way.